README.md

```diff
@@ -58,6 +58,14 @@ $ ./bin/configurer/import-ccd-definition.sh
 
 Note: Above script will export JSON content into XLSX file and upload it into instance of CCD definition store.
 
+Additional note:
+You can skip some of the files by using -e option on the import-ccd-definitions, i.e.
+```bash
```

Comment:
Please add blank line before. It will read better. Markdown should be readable well in raw and rendered form.